acpi-id.adoc

```diff
@@ -0,0 +1,30 @@
+[[acpi-ids]]
+=== RVI specific ACPI IDs
+
+ACPI ID is used in the _HID (Hardware ID), _CID (Compatibility ID) or
+_SUB (Subsystem ID) objects as described in the ACPI Specification for
+devices that do not have a standard enumeration mechanism. The ACPI ID
+consists of two parts: a Vendor ID followed by a product identifier.
+
+Vendor IDs consist of 4 characters, each character being either an
+uppercase letter (A-Z) or a numeral (0-9). The Vendor ID should be
+unique across the Industry and registered by the UEFI forum. For RVI
+standard devices, **"RSCV"** is the Vendor ID registered. Vendor-specific
+devices can use an appropriate Vendor ID registered for the manufacturer.
+
+Product Identifiers are always four-character hexadecimal numbers (0-9
+and A-F). The Device Manufacturer is responsible for assigning this
+identifier to each product model.
+
+This document contains the canonical list of ACPI IDs for the namespace
+devices that adhere to the RVI specifications. The RVI task groups may
+make pull requests against this repository to request the allocation of
+ACPI ID for any new device.
+
+[width=100%]
+[%header, cols="5,25"]
+|===
+| ACPI ID     ^| Device
+| RSCV0001     | RISC-V Platform-Level Interrupt Controller (PLIC)
```

Comment:
Can you add a section to the non-normative ACPI guidance on examples of Device (...) definitions for _both_ of these?

Reply:
Sure

Reply:
I added an example for PLIC. I didn't see need to repeat the same thing for just HID change. Let me know if I should.

Reply:
This is good.